tests/e2e/test_api.py

```diff
@@ -384,13 +380,15 @@ def test_query_call_with_improper_payload():
         assert "missing" in response.text
 
 
-def test_valid_question() -> None:
+def test_valid_question(qna_data) -> None:
```

Comment:
with your current changes (using a Q/A pair loaded from the RAG dataset), this test needs `@pytest.mark.rag`, either that or you should use a non-rag Q/A pair.